Gemfile

```diff
@@ -11,7 +11,8 @@ gem "dump", "~> 1.2", ">= 1.2.2"
 gem "google-cloud-storage", "~> 1.34", ">= 1.34.1", require: false
 gem "money-rails", "~> 1.14"
 # gem install mysql2 -v '0.5.4' -- --with-ldflags=-L/usr/local/opt/openssl/lib --with-cppflags=-I/usr/local/opt/openssl/include
-gem "mysql2", "~> 0.5.4"
+# gem install mysql2 -- --with-mysql-dir=/opt/homebrew/bin/mysql --with-mysql-lib=/opt/homebrew/Cellar/mysql/8.3.0/lib --with-mysql-include=/opt/homebrew/Cellar/mysql/8.3.0/include/mysql
```

Comment:
I will add [-v '0.5.6'] to the gem install command to ensure synchronization of the gem install and bundle install commands.
`# gem install mysql2 -v '0.5.6' -- --with-mysql-dir=/opt/homebrew/bin/mysql --with-mysql-lib=/opt/homebrew/Cellar/mysql/8.3.0/lib --with-mysql-include=/opt/homebrew/Cellar/mysql/8.3.0/include/mysql`

Reply:
did it